src/main/java/com/iotiq/user/security/AuthenticationController.java

```diff
@@ -93,4 +97,18 @@ private <T extends GrantedAuthority> PreAuthenticatedAuthenticationToken createP
         return new PreAuthenticatedAuthenticationToken(authenticationToken.getPrincipal(),
                 authenticationToken.getCredentials(), authorities);
     }
+
+    @GetMapping("/me")
+    public UserDto getCurrentUser(Principal principal) {
+        if (principal == null) throw new InvalidCredentialException();
```

Comment:
I guess, this means UNAUTHORIZED. If so, we should return HttpStatus.UNAUTHORIZED (401) with "Unauthorized access" message